packages/web/src/components/reddit-card.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import axios from "axios";
+import RedditCardDetails from "./reddit-card-details";
+import { useState, useEffect } from "react";
+import { Card, Typography } from "antd";
+const { Title } = Typography;
+interface Props {
+  height: number;
+  width: number;
+}
+const RedditCard: React.FC<Props> = (size) => {
+  const [postData, setpostData] = useState([]);
+  const [fetched, setFetched] = useState(false);
+  const data = async () => {
+    const res = await axios.get(
+      "https://www.reddit.com/r/CompanyOfHeroes/top.json?limit=50&t=week",
```

Comment:
Are we able to swap axios with fetch ? That we we don't need to include extra lib

I think code like this should work:
```
    const response = await fetch("https://www.reddit.com/r/CompanyOfHeroes/top.json?limit=50&t=week" );
   const responseData = await response.json()
   setpostData(responseData?.data?.children);
```

Fetch is modern request lib in native JS https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch 
Now it's finally also in NodeJS (but only 18+) 

So we don't need to include axios in the web projects anymore.